src/app/Icons/DeleteX.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default function DeletX(): JSX.Element {
+  return (
+    <>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="24"
+        height="24"
+        viewBox="0 0 24 24"
+        fill="var(--color-background-light)"
```

Comment:
Maybe you can give the light color the name "white"? Because, you maybe want to use it next time for a border or maybe a font or paragraph and in that case, it will look odd to write --background-light. Plus, it's not clear, whether this is a light yellow or light green or light white

Reply:
True that. I named ist `--color-green-100` :)